stories/2022-03-27_Glass_ML/ML_20th_Century.ipynb

```diff
@@ -0,0 +1,2194 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    warnings.filterwarnings('ignore')
</pre><p>Jennifer Ding:</p><ul><li>Might be good to explain the use of warnings.filterwarnings('ignore') in cell 8</li></ul><p>Joe Cerniglia:</p><p>I used the filter warning to suppress some warning I was getting in the first drafts of the code.&nbsp;I will remove the filter warning because it’s no longer applicable. Thanks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Some of the warning suppressions were still necessary but I was able to resolve some of them without warning suppression. The rest I will leave in but comment as to the reasons for the warning. I am learning that it is good practice to check the source of these warnings every so often because as certain features are replaced or modified in Python, the warning suppression may no longer solve the issue.</p>